backend/src/packages/chaiNNer_onnx/__init__.py

```diff
@@ -53,10 +39,15 @@ def get_onnx_optimizer():
         Dependency(
             display_name="ONNX",
             pypi_name="onnx",
-            version="1.13.0",
+            version="1.14.0",
             size_estimate=12 * MB,
         ),
-        *get_onnx_optimizer(),
+        Dependency(
+            display_name="ONNX Optimizer",
```

Comment:
Does it support ARM mac now?

Reply:
Yes, I think. Unless I misread the wheel list

Reply:
Confirmed it has universal wheels now